greenheart/tools/profast_tools.py

```diff
@@ -0,0 +1,288 @@
+import numpy as np
+import ProFAST
+import numpy_financial as npf
+
+
+def adjust_dollar_year(init_cost, init_dollar_year, adj_cost_year, costing_general_inflation):
```

Comment:
I'm ok with adding this function, but it should be applied in the `get_capex` function where I already put the same functionality